eng/pipelines/helix.yml

```diff
@@ -0,0 +1,33 @@
+parameters:
+  targetOS: ''
+  archGroup: ''
+  configuration: ''
+  helixQueues: ''
+  helixToken: $(BotAccount-dotnet-github-anon-kaonashi-bot-helix-token)
```

Comment:
Where is this token coming from?

Reply:
Group variable linked through the UI. I need to update it to be defined in the YAML.